keps/sig-scheduling/5007-device-attach-before-pod-scheduled/README.md

```diff
@@ -915,28 +909,43 @@ feature flags will be enabled on some API servers and not others during the
 rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
-Will consider in the beta timeframe.
+
+When this feature is enabled, if a Pod requests a resource that has
+BindingConditions, the Pod will wait in the PreBind phase until all
+BindingConditions are set to True. This means that this feature only affects the
+behavior before the Pod is scheduled, and does not affect running workloads.
+
 ###### What specific metrics should inform a rollback?
 
 <!--
 What signals should users be paying attention to when the feature is young
 that might indicate a serious problem?
 -->
-Will consider in the beta timeframe.
+
+When a timeout occurs in BindingConditions, the Pod is repeatedly re-scheduled, which leads to an increase in the `scheduler_schedule_attempts_total` metric with the label `result=unschedulable`.
```

Comment:
Can you add metrics measuring how many times a RC was scheduled with this feature, and how many times it timed out? If you see a whole bunch of timeouts, it probably means you screwed something up (like didn't deploy the controller).